docs/functions/README.md

```diff
@@ -0,0 +1,7 @@
+# Knative Functions overview
+
+--8<-- "about-functions.md"
+
+## Getting started with functions
+
+Before you can install and use Knative Functions, you must have access to a Knative development environment. To set up a development environment, you can follow the [Knative Quickstart tutorial](../getting-started/){target=_blank}.
```

Comment:
Nit: you don't need to have a knative environment running to install the `func` CLI.

```suggestion
Before you can use Knative Functions, you must have access to a Knative development environment. To set up a development environment, you can follow the [Knative Quickstart tutorial](../getting-started/){target=_blank}.
```